package.json

```diff
@@ -70,6 +70,7 @@
   ],
   "dependencies": {
     "amd-name-resolver": "0.0.5",
+    "babel-plugin-compact-reexports": "^0.1.0",
     "broccoli-babel-transpiler": "^5.6.1",
```

Comment:
I don't think we can remove that as we directly invoke it in the case that we're not using Babel@6...unless we want to drop support for that use case. Regardless, does this need to be done in this PR or could it be a follow-up task?